config/lava/chromeos/cros-boot.jinja2

```diff
@@ -13,6 +13,10 @@ tags: ['{{ device_id }}']
 {{ super () }}
 actions:
 
+{% if not cros_flash_kernel %}
+{{ kci_raise("Invalid target, CrOS parameters missing") }}
```

Comment:
I guess it'd be good to mention explicitly that the `cros_flash_kernel` section is missing

Reply:
Done